package.json

```diff
@@ -38,75 +38,74 @@
+    "@sentry/react-native": "6.5.0",
     "@tanstack/react-query": "5.62.8",
     "@turf/helpers": "7.1.0",
     "@turf/random": "7.1.0",
     "@types/luxon": "3.4.2",
     "assert": "2.1.0",
     "buffer": "6.0.3",
     "cheap-ruler": "4.0.0",
+    "debug": "4.4.0",
     "eventemitter3": "5.0.1",
-    "expo": "50.0.21",
-    "expo-application": "5.8.4",
-    "expo-av": "13.10.6",
-    "expo-build-properties": "0.11.1",
-    "expo-camera": "14.1.3",
-    "expo-crypto": "12.8.1",
-    "expo-dev-client": "3.3.12",
-    "expo-device": "5.9.4",
-    "expo-document-picker": "11.10.1",
-    "expo-file-system": "16.0.9",
-    "expo-font": "11.10.3",
-    "expo-localization": "14.8.4",
-    "expo-location": "16.5.5",
-    "expo-secure-store": "12.8.1",
-    "expo-sensors": "12.9.1",
-    "expo-splash-screen": "0.26.5",
-    "expo-status-bar": "1.11.1",
-    "expo-system-ui": "2.9.4",
-    "expo-task-manager": "11.7.3",
-    "expo-updates": "0.24.13",
+    "expo": "52.0.27",
+    "expo-application": "6.0.2",
+    "expo-av": "15.0.2",
+    "expo-build-properties": "0.13.2",
+    "expo-camera": "16.0.11",
+    "expo-crypto": "14.0.2",
+    "expo-dev-client": "5.0.10",
+    "expo-device": "7.0.2",
+    "expo-document-picker": "13.0.2",
+    "expo-file-system": "18.0.7",
+    "expo-font": "13.0.3",
+    "expo-localization": "16.0.1",
+    "expo-location": "18.0.5",
+    "expo-secure-store": "14.0.1",
+    "expo-sensors": "14.0.2",
+    "expo-splash-screen": "0.29.21",
+    "expo-status-bar": "2.0.1",
+    "expo-system-ui": "4.0.7",
+    "expo-task-manager": "12.0.4",
+    "expo-updates": "0.26.13",
     "geojson": "0.5.0",
     "geojson-geometries-lookup": "0.5.0",
     "lodash.isequal": "4.5.0",
     "luxon": "3.5.0",
     "map-obj": "5.0.2",
     "nanoid": "5.0.9",
     "nodejs-mobile-react-native": "18.20.4",
-    "react": "18.2.0",
+    "react": "18.3.1",
     "react-hook-form": "7.54.1",
     "react-intl": "7.0.4",
-    "react-native": "0.73.6",
+    "react-native": "0.76.6",
     "react-native-android-open-settings": "1.3.0",
     "react-native-confirmation-code-field": "7.4.0",
     "react-native-device-info": "14.0.1",
-    "react-native-gesture-handler": "2.21.2",
+    "react-native-gesture-handler": "2.22.0",
     "react-native-indicators": "0.17.0",
     "react-native-linear-gradient": "2.8.3",
     "react-native-mmkv": "2.12.1",
     "react-native-progress": "5.0.1",
     "react-native-reanimated": "3.16.5",
     "react-native-restart": "0.0.27",
-    "react-native-safe-area-context": "4.14.0",
+    "react-native-safe-area-context": "5.1.0",
     "react-native-scale-bar": "1.0.6",
-    "react-native-screens": "3.29.0",
+    "react-native-screens": "4.5.0",
     "react-native-share": "12.0.3",
-    "react-native-svg": "15.10.1",
+    "react-native-svg": "15.11.1",
     "react-native-url-polyfill": "2.0.0",
     "react-native-vector-icons": "10.2.0",
     "react-native-zeroconf": "0.13.8",
-    "readable-stream": "4.5.2",
```

Comment:
not entirely sure if removing this is okay but didn't seem to be necessary based on a some checks. might need to dig into the commit history a bit to know 🤔

Reply:
ah yeah added via https://github.com/digidem/comapeo-mobile/commit/6d9e242ea896c9f2559d9b4fd6b439e2ad6e895d to account for issues with how rpc-reflector was being bundled. Should no longer be needed at this point